README.md

```diff
@@ -1,10 +1,10 @@
 # ![Crayfish](https://cloud.githubusercontent.com/assets/2371345/15409657/2dfb463a-1dec-11e6-9089-06df94ef3f37.png) Crayfish
 
 [![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg?style=flat-square)](https://php.net/)
-[![Build Status](https://travis-ci.com/Islandora/Crayfish.svg?branch=main)](https://travis-ci.com/Islandora/Crayfish)
+[![Build Status](https://github.com/islandora/crayfish/actions/workflows/build-dev.yml/badge.svg)](https://github.com/Islandora/Crayfish/actions)
```

Comment:
You are not the first to do this, but I ask this question everytime. Does our `main` branch have a purpose or has the `dev` branch now just become **main** and the actual `main` branch could be turfed?

Reply:
Dev is the default branch on GitHub so that’s why I did that but I am probably wrong. @dannylamb ?

Reply:
I don't think your wrong, but if we tag a commit in `dev` as a release then I am just wondering if we even need `main` anymore? I'll merge this and we could consider removing that branch entirely. No one is working against it.